File: src/main/java/io/sn/dumortierite/registry/ItemStackRegistry.java
package io.sn.dumortierite.registry;

import org.bukkit.inventory.ItemStack;

import java.util.ArrayList;
import java.util.List;

public class ItemStackRegistry {

    private final List<ItemStack> registry = new ArrayList<>();

    public List<ItemStack> getChipsItemStack() {
        return registry.subList(0, 5);
    }

    public void init(List<ItemStack> items) {
        this.registry.addAll(items);
    }

}

File: src/main/kotlin/io/sn/dumortierite/ItemStackRegistry.kt
package io.sn.dumortierite

import org.bukkit.inventory.ItemStack

class ItemStackRegistry {

    companion object {
        lateinit var itemStackRegistry: ArrayList<ItemStack>
    }

}

val chipsItemStack: ArrayList<ItemStack> =
    ItemStackRegistry.itemStackRegistry.subList(0, 5) as ArrayList<ItemStack>
